.github/ISSUE_TEMPLATE/airflow_helmchart_bug_report.yml

```
@@ -38,33 +38,10 @@ body:
         - "main (development)"
     validations:
       required: true
-  - type: dropdown
+  - type: input
     attributes:
       label: Apache Airflow version
-      description: >
-        What Apache Airflow version are you using? Only Airflow 2 is supported for bugs. If you wish to
-        discuss Airflow 1.10, open a [discussion](https://github.com/apache/airflow/discussions) instead!
-      multiple: false
-      options:
-        - "2.3.3 (latest released)"
-        - "2.3.2"
-        - "2.3.1"
-        - "2.3.0"
-        - "2.2.5"
-        - "2.2.4"
-        - "2.2.3"
-        - "2.2.2"
-        - "2.2.1"
-        - "2.2.0"
-        - "2.1.4"
-        - "2.1.3"
-        - "2.1.2"
-        - "2.1.1"
-        - "2.1.0"
-        - "2.0.2"
-        - "2.0.1"
-        - "2.0.0"
-        - "main (development)"
+      description: What Apache Airflow version are you using? Only Airflow 2 is supported for bugs.
```

```suggestion
      description: What Apache Airflow version are you using? [Only Airflow 2 is supported](https://github.com/apache/airflow#version-life-cycle) for bugs.
```
WDYT linking to the version lifecycle section of the README in the description for both templates?